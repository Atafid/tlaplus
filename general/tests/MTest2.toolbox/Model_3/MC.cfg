SPECIFICATION spec_12495109754386000
\* PROPERTY definition
PROPERTY
prop_12495109754387000
\* Generated at Wed Aug 05 15:22:55 PDT 2009