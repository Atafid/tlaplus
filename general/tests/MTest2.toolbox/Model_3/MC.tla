---- MODULE MC ----
EXTENDS MTest2, TLC

\* Specification formula
spec_12495109754386000 ==
(x=0) /\ [][x'=x]_x
----
\* PROPERTY definition
prop_12495109754387000 ==
[]<>[][x'=0]_x
----
====
\* Generated at Wed Aug 05 15:22:55 PDT 2009