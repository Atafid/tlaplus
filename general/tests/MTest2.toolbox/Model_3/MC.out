TLC2 Version 2.02 of 3 August 2009
@!@!@STARTMSG 2187:0 @!@!@
Running in Model-Checking mode.
@!@!@ENDMSG 2187 @!@!@
@!@!@STARTMSG 2220:0 @!@!@
Starting SANY...
@!@!@ENDMSG 2220 @!@!@
Parsing file MC.tla
Parsing file MTest2.tla
Parsing file C:\users\lamport\tla\tla2\org.lamport.tla.toolbox\StandardModules\TLC.tla
Parsing file C:\users\lamport\tla\tla2\org.lamport.tla.toolbox\StandardModules\Naturals.tla
Parsing file C:\users\lamport\tla\tla2\org.lamport.tla.toolbox\StandardModules\Sequences.tla
Semantic processing of module MTest2
Semantic processing of module Naturals
Semantic processing of module Sequences
Semantic processing of module TLC
Semantic processing of module MC
@!@!@STARTMSG 2219:0 @!@!@
SANY finished.
@!@!@ENDMSG 2219 @!@!@
@!@!@STARTMSG 2185:0 @!@!@
Starting...
@!@!@ENDMSG 2185 @!@!@
@!@!@STARTMSG 1000:1 @!@!@
@!@!@STARTMSG 2214:0 @!@!@
Temporal formulas containing actions must be of forms <>[]A or []<>A.
@!@!@ENDMSG 2214 @!@!@
@!@!@ENDMSG 1000 @!@!@
